### ISSUE TITLE
step or stl files for the case

### ISSUE THREAD

**Issue by brunnels** [eleclab-rpi/5inch_480x854#2]
Can you please provide the step or stl files for the case?

This screen works great with klipperscreen for my 3d printer and I would like to create a mount for the display on my printer.  It would be much easier to start from the existing files than to start from scratch.

https://klipper.discourse.group/t/hardware-known-to-work-with-klipperscreen/35/24?u=kraven

**Comment by eleclab-rpi** [comment]
Oh, you mean STL with 5" DPI 480x854 display? Only the front case and the speaker back cover are now, the overall back cover has not been designed yet. We didn't decide which casing and cooling structure to use before, now we are ready Use the turbo cooling fan like a notebook, so we will have to wait a few days for our turbo sample to arrive and take pictures for you to see.

The stl file has been uploaded to github:   https://github.com/eleclab-rpi/5INCH_DPI_DISPLAY



***@***.***
 
From: Benjamin Runnels
Date: 2022-05-14 05:33
To: eleclab-rpi/5inch_480x854
CC: Subscribed
Subject: [eleclab-rpi/5inch_480x854] step or stl files for the case (Issue #2)
Can you please provide the step or stl files for the case?
This screen works great with klipperscreen for my 3d printer and I would like to create a mount for the display on my printer. It would be much easier to start from the existing files than to start from scratch.
https://klipper.discourse.group/t/hardware-known-to-work-with-klipperscreen/35/24?u=kraven

**Comment by brunnels** [comment]
Thank you that should help.  I just took mine apart.  What is the best way remove the top screen from the resin printed frame?  It looks like it's glued in?

**Comment by brunnels** [comment]
I got it off.  I converted the stl's to step files and added to my fork.  I also added my first draft for an enclosure with rpi connected to gpio via ribbon cable rather than mounted to the screen.  https://github.com/brunnels/5inch_480x854/blob/main/5_INCH_SCREEN_ENCLOSURE%20v1.step

**Comment by eleclab-rpi** [comment]
Yes, the touchscreen is glued on, while the LCD is stuck through the case.



***@***.***
 
From: Benjamin Runnels
Date: 2022-05-14 20:02
To: eleclab-rpi/5inch_480x854
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/5inch_480x854] step or stl files for the case (Issue #2)
Thank you that should help. I just took mine apart. What is the best way remove the top screen from the resin printed frame? It looks like it's glued in?

**Comment by eleclab-rpi** [comment]
Great job! You're really fast, but the RPI's 40P XH2.54 plug is already plugged into the display PCB.



***@***.***
 
From: Benjamin Runnels
Date: 2022-05-15 04:42
To: eleclab-rpi/5inch_480x854
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/5inch_480x854] step or stl files for the case (Issue #2)
I got it off. I converted the stl's to step files and added to my fork. I also added my first draft for an enclosure with rpi connected to gpio via ribbon cable rather than mounted to the screen. https://github.com/brunnels/5inch_480x854/blob/main/5_INCH_SCREEN_ENCLOSURE%20v1.step

**Comment by brunnels** [comment]
I'm going to use an extension cable like this https://www.adafruit.com/product/4823

**Comment by eleclab-rpi** [comment]
I understand your usage, so do you need to use the signal in XH2.54 40PIN?



***@***.***
 
From: Benjamin Runnels
Date: 2022-05-15 19:59
To: eleclab-rpi/5inch_480x854
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/5inch_480x854] step or stl files for the case (Issue #2)
I'm going to use an extension cable like this https://www.adafruit.com/product/4823

**Comment by brunnels** [comment]
yeah, I just want the panel mounted on the front of the printer and the RPI 20cm away mounted on a DIN rail under the printer.  I'll send you my full printer step when I'm done.  Based on VzBOT with RatRig V-Core triple Z, and voron 2.4 inspired skirt and electronics mounted under the printer.

**Comment by eleclab-rpi** [comment]
Oh, great job. You can take a nice photo if the prototype comes out. I'd love to add your idea to github if you allow it.



***@***.***
 
From: Benjamin Runnels
Date: 2022-05-16 01:57
To: eleclab-rpi/5inch_480x854
CC: ElecLab; Comment
Subject: Re: [eleclab-rpi/5inch_480x854] step or stl files for the case (Issue #2)
yeah, I just want the panel mounted on the front of the printer and the RPI 20cm away mounted on a DIN rail under the printer. I'll send you my full printer step when I'm done. Based on VzBOT with RatRig V-Core triple Z, and voron 2.4 inspired skirt and electronics mounted under the printer.
